lib/cdo/aws/rds.rb

```diff
@@ -0,0 +1,120 @@
+require_relative '../../../deployment'
+require 'aws-sdk-rds'
+
+module Cdo
+  class RDS
+    # Create an RDS Aurora MySQL cluster cloned from a specified source cluster.
+    # @param source_cluster_id [String] DB cluster id of the cluster to clone.  Defaults to current environment's cluster.
+    # @param clone_cluster_id [String] DB cluster id to assign to clone.  Defaults to source cluster id + "-clone"
+    # @param instance_type [String]
+    def self.clone_cluster(
```

Comment:
I chose to leave these as class methods.  Cluster clones are typically long lived.  I think it's rare that a caller is going to instantiate `Cluster`, call then clone method on it, and then later call the delete method on it.

Reply:
Just to clarify my suggestion wasn't about assuming a long-lived object lifecycle, I just thought it would result in more compact/simple code by removing some duplication across the class methods. Fine to leave as-is though, thanks for considering.

Reply:
True.  I agree that the implementation of these 2 methods would be more compact.  I'm thinking that usage of these methods would be less compact (instantiate + invoke method).

Reply:
Ah, I understand now, yeah that's a good point.